applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -93,33 +98,27 @@ Forwarding HID input reports
 
 After :ref:`nrf_desktop_ble_discovery` successfully discovers a connected peripheral, the |hid_forward| automatically subscribes for every HID input report provided by the peripheral.
 The subscriber can use either the HID boot protocol or the HID report protocol, but both protocols cannot be used at the same time.
-In the current implementation, the :ref:`nrf_desktop_usb_state` supports either the HID boot keyboard or the HID boot mouse reports, because the HID boot protocol code is set using a single Kconfig option that is common for all of the instances of the USB HID.
+In the current implementation, the :ref:`nrf_desktop_usb_state` supports either the HID boot keyboard or the HID boot mouse reports, because the application does not support assigning HID boot protocol code separately for each USB HID instance.
 
 The :c:func:`hogp_read` callback is called when HID input report is received from the connected peripheral.
-The received HID input report data is converted to ``hid_report_event``.
+The received HID input report data is added to the HID report queue and converted to a :c:struct:`hid_report_event`.
```

Comment:
` the HID report queue utility` - we might not add it to queue

Reply:
`is added to` -> `is passed to`